### ISSUE TITLE
Genes not included in gene panel versions in use when loading this case (clinical list):['TAP1', 'TAP2']

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#3024]
The **express** case ([421701400259fam](https://scout.scilifelab.se/cust004/421701400259fam)) was analysed with [PID](https://scout.scilifelab.se/panels/603f92c9d05a2282ecd74c50) (Primary Immune deficiency) as default panel. This panel contains the genes TAP1 and TAP2, but if you try to filter for variants in these 2 genes it complains:

![image](https://user-images.githubusercontent.com/28093618/144983311-a833535d-9281-4f1f-a4d9-eb48a1de780b.png)

WHY?

**Comment by northwestwitch** [comment]
Mmm we might have a bug in variantS filters here.

**Comment by dnil** [comment]
Hm, probably to do with both of the symbols also being aliases for other hgnc genes. 

![Screenshot 2021-12-07 at 08 34 27](https://user-images.githubusercontent.com/758570/144986179-6d015844-7d43-41bb-9548-eac120cae6b3.png)
![Screenshot 2021-12-07 at 08 34 32](https://user-images.githubusercontent.com/758570/144986182-fd6785f1-fee7-4233-94bb-e0827f7dd954.png)

**Comment by dnil** [comment]
Maybe the prio for alias and symbol was messed up at some point, perhaps when adding the caption genes?

**Comment by northwestwitch** [comment]
I'll try to reproduce it locally, let's see

**Comment by northwestwitch** [comment]
In the meanwhile they found the causative variant in another gene, so at least it's not THAT urgent, but let's fix it!

**Comment by dnil** [comment]
Good. I feel it is most likely an error with the warning rather than with the filter as such, but I would not have said that if it was still an open case.

**Comment by northwestwitch** [comment]
No, but I've loaded all variants in those 2 genes for this case, so the filters should return something I think

**Comment by northwestwitch** [comment]
For instance, this variant here is in one of those genes: https://scout.scilifelab.se/cust002/F0038718-2/c931ac63ccbdc71709d8c899d6897fad. Try to search the variant using filters

**Comment by northwestwitch** [comment]
Could also have been when I changed the query filters here: https://github.com/Clinical-Genomics/scout/commit/2489166c54816ac65371304d9cd34f9a86d3fbc2

**Comment by dnil** [comment]
All TAP1 variants in the vcf were negative score, but load all should have brought it in.

**Comment by dnil** [comment]
As a corollary, searching for an alias to one of the TAP1, TAP2 also apparently gives another unexpected warning:
![Screenshot 2021-12-07 at 08 56 28](https://user-images.githubusercontent.com/758570/144989241-658adf71-0c6e-429e-9da3-7e6c1b4d4dfa.png)
One would have liked the aliasing to work there as well.
